models/migrations/v1_20/v255.go

```diff
@@ -0,0 +1,32 @@
+// Copyright 2023 The Gitea Authors. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+package v1_20 //nolint
+
+import (
+	"code.gitea.io/gitea/modules/timeutil"
+
+	"xorm.io/xorm"
+)
+
+func CreateActionArtifactTable(x *xorm.Engine) error {
+	type ActionArtifact struct {
+		ID               int64 `xorm:"pk autoincr"`
+		JobID            int64 `xorm:"index"`
```

Comment:
It should be kept as the same as the defenition in the `models/actions` package.